Graphics/GraphicsEngine/interface/GraphicsTypes.h

```diff
@@ -35,6 +35,7 @@
 #include "../../../Primitives/interface/BasicTypes.h"
 #include "../../../Primitives/interface/DebugOutput.h"
 #include "../../../Primitives/interface/FlagEnum.h"
+#include "../../../Platforms/interface/NativeWindowHandle.hpp"
```

Comment:
`.h` and `.hpp` extensions are used to clearly indicate which files can be used in both c and c++, and which are c++ only. Native window handle will be part of the core API and is compatible with both c and c++, so it should have .h extension